crates/gosub_cairo/src/debug/text.rs

```diff
@@ -0,0 +1,92 @@
+use crate::elements::brush::GsBrush;
+use crate::elements::color::GsColor;
+use crate::elements::transform::GsTransform;
+use crate::Scene;
+use gosub_shared::render_backend::{Brush as _, Color as _, Transform as _};
+use gosub_shared::types::Point;
+use gosub_typeface::ROBOTO_FONT;
+use peniko::{Blob, Font};
+use std::sync::{Arc, LazyLock};
+
+static FONT: LazyLock<Font> = LazyLock::new(|| Font::new(Blob::new(Arc::new(ROBOTO_FONT)), 0));
```

Comment:
I guess this can be removed, since we can't use this with cairo

Reply:
FONT is being used in the render_text(). For now it doesn't do much though